### DISCUSSION TITLE
Why is train_kuka_grasping stuck?

### DISCUSSION THREAD

**Discussion by HaoyuanXu** [bulletphysics/bullet3#3704] [General]
2020-07-20 16:10:52.857284: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that thi
s TensorFlow binary was not compiled to use: AVX2
WARNING:tensorflow:From D:\anaconda\envs\bullet\lib\site-packages\tensorflow\python\framework\op_def_library.py:263: col
ocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
WARNING:tensorflow:From D:\anaconda\envs\bullet\lib\site-packages\tensorflow\contrib\layers\python\layers\layers.py:1624
: flatten (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.flatten instead.
WARNING:tensorflow:From D:\anaconda\envs\bullet\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from
 tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
D:\anaconda\envs\bullet\lib\site-packages\numpy\core\fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
D:\anaconda\envs\bullet\lib\site-packages\numpy\core\_methods.py:161: RuntimeWarning: invalid value encountered in doubl
e_scalars
  ret = ret.dtype.type(ret / rcount)

And it is stuck then  ,How can I solve it? Thanks in advance/

By the way, does anyone using a sparse reward  or  reward relating to distance?
I have seen the issue  'how much time for train kuka grasping? #1386'  And I have the same issue...
